project-base/storefront/components/Pages/CategoryDetail/CategoryDetailContent.tsx

```diff
@@ -23,10 +24,12 @@ type CategoryDetailContentProps = {
 export const CategoryDetailContent: FC<CategoryDetailContentProps> = ({ category }) => {
     const t = useTypedTranslationFunction();
     const [isPanelOpen, setIsPanelOpen] = useState(false);
-    const containerWrapRef = useRef<null | HTMLDivElement>(null);
+    const paginationScrollTargetRef = useRef<HTMLDivElement>(null);
```

Comment:
Is the `paginationScrollTargetRef` different than the previously used `containerWrapperRef`? And the question regards the entire PR, not just here...

Reply:
Nope, I was following implementation from Tescoma since I have found it better descriptive.